### PR TITLE
instructorFeedbacksPage - rename 'Course' --> 'Course ID' #2955

### LINKED ISSUE #2955: instructorFeedbacksPage - rename 'Course' --> 'Course ID'
The dropdown when adding a session contains course ids, but the label is "Course"
Renaming the label/including the course names in the dropdown might make it clearer.

Other places in teammates may have this issue as well.

image

### PR DESCRIPTION
**Pull request by arvind-kalyan**
Fixes #2955

### CODE REVIEW

**Comment by damithc** [comment]
@kanghj for your review please

File: src/test/resources/pages/godmode.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
             type="text"
```
Comment (kanghj) [inline src/test/resources/pages/godmode.html]: You should exclude changes to this file as this is unrelated to the issue

File: src/test/resources/test.template.properties [hunk 2]
```diff
@@ -33,7 +33,7 @@ test.selenium.host=localhost
 # To use default firefox setting, leave the field as is,
 # to use custom path, e.g. 
 #test.firefox.path=C:\\Program Files (x86)\\Mozilla Firefox 12\\firefox.exe
-test.firefox.path=
+test.firefox.path=C:\\Program Files (x86)\\Mozilla Firefox 21\\firefox.exe
```
Comment (kanghj) [inline src/test/resources/test.template.properties]: You should exclude changes to this file as this is unrelated to the issue.

**Comment by kanghj** [comment]
Some changes requested. The other changes look good.

**Comment by arvind-kalyan** [comment]
Thanks. I will fix it in around 3-4 hours and comment here.

**Comment by arvind-kalyan** [comment]
changes made. kindly check

**Comment by damithc** [comment]
@arvind-kalyan please double check the files in the patch. There are unrelated changes. @kanghj is on holiday (for Chinese New Year). he will check your code in a few days.

**Comment by kanghj** [comment]
There are still changes to test.template.properties that shouldn't be there. You can check which files have changes under the "Files changed" tab on this pull request.

**Comment by arvind-kalyan** [comment]
Hello, I'm having some trouble understanding how to avoid those files from getting commited, as I'm a beginner to git. I reset to a prev commit using git reset --hard **f2821a4**. So my HEAD is now at 	: **InstructorHelp.html file modified accordingly. Also fixed one grammat… …	 f2821a4**. How do I avoid the two files, godmode.html and test.template.properties from getting added? Please help!

**Comment by damithc** [comment]
If you are on Windows or OS-X, you can use SoureTree (A Git Client). It
allows you to select which files are to be committed.

**Comment by kanghj** [comment]
Yes, this is correct. 

Looks good to merge.

**Comment by damithc** [comment]
Go ahead and merge @kanghj
